### ISSUE TITLE
Autoscan not working - Manual scan do

### ISSUE THREAD

**Issue by Sephirot171** [a1ex4/ownfoil#157]
Hello,
  I'm running Ownfoil on docker on a VM with Debian 12, I'm using the "develop" image.
The llibrary path on the VM is a directory mounted via NFS3 from another server.
The games/dlcs/updates are working correctly and when I manually scan the library everityng new is recognized, meanwhile the auto scan of new things is not.
I've tried waiting for 24h but the autoscan still didn't pick up the new objects.
Where can i check what's not working correctly?

Thanks.
Have a nice day.

**Comment by a1ex4** [comment]
Hi, Ownfoil is using the Python watchdog to monitor for change events in the filesystem, my guess is that it is not working for mounted volumes. You could test on your system with a simple script to see if the library indeed does not see the changes happening, but I am not sure what can be done otherwise.

In the future I will also add periodic scans of the library on top of filesystem monitoring, but for the time being you will have to trigger the scan manually.

**Comment by Sephirot171** [comment]
Hi Alex, thanks for the answer. Can this https://github.com/gorakhargosh/watchdog/issues/504 make it work?

**Comment by Sephirot171** [comment]
Hi Alex, sorry for the delay. I tested now and it's working correctly, AWESOME! Thank you very much!

**Comment by dakky** [comment]
is this change already in the latest docker-image? I have the same issue: only a mnual refresh shows my library

**Comment by Sephirot171** [comment]
Hi Alex, is seems like it doesn't work anymore.

**Comment by a1ex4** [comment]
> Hi Alex, is seems like it doesn't work anymore.

What's not working anymore?

**Comment by Sephirot171** [comment]
Sorry for the delay, the autoscan on NFS looked like not working, but I tried just now with an update and it worked again.
Maybe for new games it doesn't work, I'll keep you posted.

Have a nice day.

**Comment by Sephirot171** [comment]
ok tried.
Downloaded update file with browser and put on the share it DOES work, downloaded directly on the folder via torrent DOESN'T.
Could it be that the torrent puts ".part" on the end of the filename before completion?

Have a nice day.

**Comment by a1ex4** [comment]
>  Could it be that the torrent puts ".part" on the end of the filename before completion?

Yes I think that was the reason, I pushed a fix to the `develop` image that you can test, let me know if that fixes your issues.

**Comment by Sephirot171** [comment]
works perfectly, thanks!
